lazy.js

```diff
@@ -6628,6 +6628,37 @@
     return snapshot;
   };
 
+  Sequence.prototype.flatten = function flatten () {
```

Comment:
This method already exists. If the goal is to add a non-recursive `flatten` method per #125, I think the answer is to add support for a `depth` flag or something like that to `FlattenedSequence`, not to start from scratch.